playbooks/roles/selinux/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: 'Ensure libselinux-python is installed'
+  yum:
+    name: 'libselinux-python'
+    state: installed
```

Comment:
And could help ensure SELinux is enforcing by default in case the box someone uses has it disabled for any reason